tests/Benchmark/Persistence/PersistentFactoryBench.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Zenstruck\Foundry\Tests\Benchmark\Persistence;
+
+use PhpBench\Attributes\BeforeClassMethods;
+use PhpBench\Attributes\BeforeMethods;
+use PhpBench\Attributes\ParamProviders;
+use Zenstruck\Foundry\Persistence\PersistentObjectFactory;
+use Zenstruck\Foundry\Tests\Benchmark\KernelBench;
+
+#[BeforeClassMethods(['_resetDatabaseBeforeFirstBench'])]
+#[BeforeMethods(['_bootFoundry', '_resetDatabaseBeforeEachBench'])]
+abstract class PersistentFactoryBench extends KernelBench
```

Comment:
I had to split into two different base classes, because, performing all these `static::factory()->many($params['count'])->create();` in `_setup_bench_random()` was really time consuming